tools/rdock/rdock.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="rdock" name="rDock" version="1.0">
+    <description>Binding Mode Prediction in Proteins/RNA</description>

```

Comment:
A <requirement> should be added here, let me know if you need a conda package as well.